lib/src/core/better_player_controller.dart

```diff
@@ -627,6 +631,20 @@ class BetterPlayerController {
     }
   }
 
+  ///Show skipIntroButton
+  Future<void> showSkipIntroButton() async {
+    _showSkipIntro = true;
```

Comment:
w sumie co to za flaga - czy to nie powinno sie jakos inaczej nazywac? A jak nie, to czy na pewno tutaj powinismy ta flage uzyc

Reply:
Tą flage dałem by z kontrolera sprawdzać czy ma się pokazać skipIntro czy nie, zroviłem to na podstawie jak _isFullScreen